octobot_tentacles_manager/api/inspector.py

```diff
@@ -65,3 +79,63 @@ def check_tentacle_version(version, name, origin_package, verbose=True) -> bool:
         if verbose:
             logger.error(f"Error when reading tentacle metadata: {e}")
     return True
+
+
+def _load_tentacle_class(tentacle_name):
```

Comment:
Why do we need to do that?

Reply:
It's used in `get_tentacle_class_from_string` which is used many times in scripts as well as for the web interface. As we start to have many types of tentacles, I think it's better to group this code at one place instead of in many different places. Moreover introspection takes time and if we can cache it, it's much better